REQUIREMENTS.md

```diff
@@ -0,0 +1,16 @@
+# Argo Certification Program Requirements
+
+## Certification is provided for each of the four Argo projects.
+* [Argo Workflows](https://github.com/argoproj/argo-workflows)
+* [Argo Events](https://github.com/argoproj/argo-events/)
+* [Argo CD](https://github.com/argoproj/argo-cd/)
+* [Argo Rollouts](https://github.com/argoproj/argo-rollouts/)
+
+## For any certification the following is required
+* Offering provided by CNCF member company
+* Offering provisions, manages, or hosts compliant instances of target project
+* Member company maintains X maintainer(s) within the Argo org. See [Maintainers](https://github.com/argoproj/argoproj/blob/master/MAINTAINERS.md)
+* Confirm through self-testing the offering is compatible with target project version
+* Provider will maintain offering with latest versions within X months of project release.
+* Offering must include option for accessing the default UI with only minor modificaitons.
```

Comment:
We could say without modifications since there is an extensions system for people to potentially add UI elements. And I agree, they could have a separate UI aswell.

Reply:
Agreement from @hblixt @wanghong230 @wtam2018 @jannfis on using latest 2 versions rather than. timeframe